### PR TITLE
Updated learn section-2

### PR DESCRIPTION
**Pull request by AanshSavla**
## Changes
Solves issue https://github.com/Qiskit/qiskit.org/issues/3071
Added all the missing content in top level drop down menu and missing content in other pages.
Solves issue of PR: #3109

### CODE REVIEW

File: content/ecosystem/members.json [hunk 1]
```diff
@@ -1,54 +1,128 @@
 [
   {
-    "name": "bosonic-qiskit",
-    "url": "https://github.com/C2QA/bosonic-qiskit",
-    "description": "NQI C2QA project to simulate hybrid boson-qubit systems within Qiskit.",
-    "licence": "BSD 2-Clause Simplified or FreeBSD license",
+    "alternatives": null,
```
Comment (techtolentino) [inline content/ecosystem/members.json:3]: As this is not related to your PR, please ensure that it's not included.

Thanks @AanshSavla

Comment (AanshSavla) [reply content/ecosystem/members.json]: I have removed the members.json changes.

**Comment by AanshSavla** [comment]
I have removed the changes by members.json file.

**Review by frankharkins — APPROVED** [review]
Looks good, thanks!

**Comment by AanshSavla** [comment]
I have a question. If this branch is merged with main why are the changes not reflected on the website?

**Comment by frankharkins** [comment]
@AanshSavla `Merge branch 'main' into learn_update-branch` means your branch has been updated with the latest version of `main`. We do this to check recent changes to `main` don't conflict with changes in this pull request. The `main` branch has not been changed yet – that will happen when this pull request is merged.

After `main` is updated, the actual website won't be updated until the [deploy workflow](https://github.com/Qiskit/qiskit.org/blob/main/.github/workflows/deploy.yml) runs (each day @ 8am).

@techtolentino correct me if I've got any of that wrong.

**Comment by AanshSavla** [comment]
Okay. Thanks for this information!

**Comment by AanshSavla** [comment]
Also now if I want to work on some other issue then should I make a branch from the main branch or the learn_update-branch ?

**Comment by techtolentino** [comment]
@AanshSavla,

Thanks for asking this question. @frankharkins is right, this was done just to keep _your_ branch up to date. 

And for the new branch question: yes, please start a new branch based on `main`, anytime you consider a new issue. This helps in trying to keep the code focused on solving a single issue; it makes it easier to review, and quicker to merge.
